subsys/dult/near_owner_state.c

```diff
@@ -35,4 +47,5 @@ enum dult_near_owner_state_mode dult_near_owner_state_get(void)
 void dult_near_owner_state_reset(void)
 {
 	cur_mode = DULT_NEAR_OWNER_STATE_MODE_NEAR_OWNER;
+	state_cb = NULL;
```

Comment:
We assumed that internal callbacks within the DULT module would be sticky (they don't get cleared during the `dult_reset`). You can take a look at DULT sound <-> BT ANOS or DULT sound <-> MD callback interaction for reference

Reply:
Yes, you're right